CHANGELOG.md

```diff
@@ -7,6 +7,94 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
+- [#30] added a `--no-flash` flag to run a program without re-flashing it
+
+[#30]: https://github.com/knurling-rs/probe-run/pull/30
+
+- [#40] added (`--help`) documentation to the many CLI flags
+
+[#40]: https://github.com/knurling-rs/probe-run/pull/40
```

Comment:
Interleaving them like this results in more spacing between the entries. Could you either update the new entries to not do that, or make the old entries have the same spacing?

Reply:
I wrote them like this because it make it easier to not miss adding the `[#12]: https` line. Unfortunately, it seems the link doesn't render properly if there's no newline between the bullet and the outline link. I can cluster the links again.